public/js/gulliver.es6.js

```diff
@@ -89,10 +90,13 @@ class Gulliver {
     this._addRoute(/\/pwas\/add/, transitionStrategy, setupPwaForm, {
       showTabs: false,
       backlink: true,
-      subtitle: true
+      subtitle: true,
+      search: false
     });
 
     const setupCharts = () => {
+      // Clear search-input value
+      document.querySelector('#search-input').value = '';
```

Comment:
Is there a special reason why we are setting the value of `#search-input` inside the `setupCharts` method?